### PR TITLE
Fix examine text for stackable untradables to not show a GE value 

### PR DESCRIPTION
**Pull request by TylerNichols**
Fixes issue #5841

### CODE REVIEW

**Comment by deathbeam** [comment]
You should maybe do the same thing with alch price as well

**Comment by TylerNichols** [comment]
Are there items with 0 gp alch values? I do agree the code should be formatted that way though because logically it makes more sense.

**Comment by raiyni** [comment]
Bones

File: runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java [hunk 1]
```diff
@@ -339,19 +339,20 @@ private void getItemPrice(int id, ItemComposition itemComposition, int quantity)
...
+						.append(ChatColorType.NORMAL)
+						.append(" (")
+						.append(ChatColorType.HIGHLIGHT)
+						.append(StackFormatter.formatNumber(alchPrice))
+						.append(ChatColorType.NORMAL)
+						.append("ea)");
+				}
 			}
 
+
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/examine/ExaminePlugin.java]: Remove this empty line

**Pull request merged**